src/modules/jobs/index.spec.ts

```diff
@@ -1,29 +1,37 @@
 import { Repository } from 'typeorm';
-import { Job } from '@/common/entities/Job.entity';
+import { Request } from '@entities/Request.entity';
+import { Conversation } from '@entities/Conversation.entity';
+import { Job } from '@entities/Job.entity';
 import { JobsController } from './job.controller';
 import { JobsService } from './job.service';
-import { UserService } from '../user/user.service';
+import getJobByIdResponseDto from './dto/get-job-response.dto';
 
 describe('JobsController', () => {
   let jobsController: JobsController;
   let jobsService: JobsService;
-  let userService: UserService;
-  let jobRepo: Repository<Job>;
+
+  let jobRepository: Repository<Job>;
+  let requestRepository: Repository<Request>;
+  let conversationRepository: Repository<Conversation>;
 
   beforeEach(async () => {
-    jobsService = new JobsService(userService, jobRepo);
+    jobsService = new JobsService(
+      jobRepository,
+      requestRepository,
+      conversationRepository,
+    );
     jobsController = new JobsController(jobsService);
   });
 
-  describe('getOneJob', () => {
+  describe('getJobById', () => {
     it('should return job by id', async (): Promise<void> => {
```

Comment:
at least one more test case came to my mind, what if id not exist

Reply:
@lyudad id is validating by 'class-validator' before calling the controller. So, I created dto.spec.ts for testing dto